.github/workflows/main.yml

```
@@ -38,21 +38,23 @@ jobs:
       - name: Build project
         run: yarn build
 
-      - name: SHOW AWS CLI VERSION
-        env:
-          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
-          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-          AWS_EC2_METADATA_DISABLED: true
-        run: |
-          aws --version
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
```

actionlint

the runner of "aws-actions/configure-aws-credentials@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)